app/Http/Controllers/API/Auth/RegisteredUserController.php

```
@@ -0,0 +1,46 @@
+     */
+    public function __construct(StatefulGuard $guard)
+    {
+        $this->guard = $guard;
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     */
+    public function store(Request $request, CreateNewUser $creator): string
+    {
+        if (config('fortify.lowercase_usernames') && $request->has(Fortify::username())) {
```

PHPMD (2.15.0)

Avoid using static access to class '\Laravel\Fortify\Fortify' in method 'store'. (Clean Code Rules)

(StaticAccess)